packages/client/src/views/GrantDetailsView.vue

```diff
@@ -29,6 +29,65 @@
           {{ currentGrant.title }}
         </h2>
 
+        <!-- Left page column: main print/copy/grants.gov buttons -->
+        <div class="grant-details-main-actions print-d-none">
+          <div class="d-flex justify-content-between align-items-center">
+            <b-button
+              variant="primary"
+              block
+              class="mr-5"
+              style="width: fit-content; white-space: nowrap; "
```

Comment:
A cleaner way to achieve the new layout, using existing bootstrap helpers: 

1. Remove the `block` option (instead of the `width: fit-content` style)
2. Add the `text-nowrap` utility class (instead of the `white-space: nowrap` style)
3. You can remove the `mr-5` class now that we don't need a right margin on the button

Reply:
<img width="400" alt="image" src="https://github.com/usdigitalresponse/usdr-gost/assets/13317264/14785a3d-405d-4ed8-a266-f9e9d28d6750">

I left a margin to the right of the button to add some negative space between "view" and "copy" in the case of the minimum viewport width - this wasn't really specified in the designs though. Let me know what you think

Reply:
Good call! Sorry, I just assumed it had been carried over without intention from the previous arrangement, my bad. 

Longer-term, I know the team is looking at supporting mobile designs on smaller screen sizes, so we might be able to address this sort of thing even more robustly...